File: Vector4.java
import java.util.Vector;
import java.util.Scanner;

public class Vector4{
	public static void main(String args[]){
		Scanner sc = new Scanner(System.in);
		Vector<Integer> xs = new Vector<>(7);
		
		System.out.print("First integer: ");
		int x = sc.nextInt();

		xs.add(x);

		for(int i=1; i<7; i++){
			x++;
			xs.add(x);
		}

		System.out.print("{");
		for(int i=0; i<7; i++){
			System.out.print(xs.get(i) + ", ");
		}
		System.out.println("}");

		sc.close();
	}
}

File: vector4.cpp
#include <iostream>
#include <vector>

using namespace std; 

int main(){
	vector<int> xs(7);
	int x=0;

	cout << "First integer: ";
	cin >> x;
	xs[0] = x;

	for(int i=1; i<7; i++){
		x++;
		xs[i] = x;
	}
	
	cout << "{";
	for(int i=0; i<7; i++){
		cout << xs.at(i) << ", ";
	}
	cout << "}";

	cout << endl;
	return 0;
}
